Examples/SCNVector3+Magic.swift

```diff
@@ -0,0 +1,131 @@
+//
```

Comment:
Can we rename this or add a description other than "magic"?

Reply:
Good point. The file came from something I found searching for SCNVector & Bezier helpers. We should rename the file and add a reference to the source: https://github.com/NathanFlurry/SceneKit-Splines